### PR TITLE
Dependency on current major version of cohere

### PR DESCRIPTION
**Pull request by egrefen**
### What this PR does

This introduces a dependency on the 3.x cohere version, for compatibility with the current major release.

### How it was tested

Ran poetry on an external project depending on this branch. Works fine.

### PR checklist

- [X] No API keys or other secrets committed to source?
- [X] New functionality is added alongside appropriate tests?
- [X] All source files have the following header?

```
# Copyright (c) {YEAR} Cohere Inc. and its affiliates.
#
# Licensed under the MIT License (the "License");
# you may not use this file except in compliance with the License.
#
# You may obtain a copy of the License in the LICENSE file at the top
# level of this repository.
```

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -8,7 +8,7 @@ authors = ["Cohere ConvAI <convai@cohere.com>"]
 
 [tool.poetry.dependencies]
 python = "^3.8"
-cohere = "^2.8.0"
+cohere = ">=2.8.0"
```
[the remark this answers is not in the document]
Comment (egrefen) [reply pyproject.toml:11]: Okay will do.

[the remark this answers is not in the document]
Comment (egrefen) [reply pyproject.toml]: Done.

[the remark this answers is not in the document]
Comment (sanderland) [reply pyproject.toml]: Probably want `version = "0.2.0"` for this package as well, though it's a bit of a technicality.

**Review by sanderland — APPROVED** [review]
will leave to @eddotman et al. to confirm, but lgtm
-> actually, tests fail, will investigate

**Review by sanderland — APPROVED** [review]

File: tests/conftest.py [hunk 2]
```diff
@@ -26,9 +26,10 @@
 
 
 class MockCo:
+
     def generate(*args, **kwargs) -> Generations:
         return Generations(
-            response={"generations": [dict(text="Hello!", likelihood=1.0)]},
+            response={"generations": [dict(text="Hello!", likelihood=1.0, id=123)]},
```
Comment (eddotman) [inline tests/conftest.py:32]: thank you for catching this 🙏

**Review by eddotman — APPROVED** [review]
yeah also LGTM!

**Comment by yichern** [comment]
Added an item in our changelog too

**Pull request merged**